app/views/users/_form.html.erb

```diff
@@ -1,6 +1,16 @@
 <%= simple_form_for(@user) do |f| %>
   <%= f.input :introduction, label: 'Introduction', placeholder: 'Some words about yourself' %>
   <%= f.input :goal_description, label: 'Describe your goal'  %>
```

Comment:
Could you please put: placeholder: 'I want to achieve this goal because...'

Reply:
I have not seen your comment but wrote exactly the same phrase!

Reply:
done